### PR TITLE
PLATFORM-1955: unify logs severity level

### PR DESCRIPTION
**Pull request by macbre**
[PLATFORM-1955](https://wikia-inc.atlassian.net/browse/PLATFORM-1955)

Unify severity of error messages sent by MediaWiki
- replace `Wikia::log` calls with `WikiaLogger` + proper severity
- remove `Wikia::log` calls that duplicates `WikiaLogger` messages

@wladekb

### CODE REVIEW

File: extensions/wikia/ArticleComments/classes/ArticleCommentList.class.php [hunk 1]
```diff
@@ -776,8 +776,6 @@ static public function articleDeleteComplete( WikiPage &$wikiPage, &$user, $reas
 						$task = new \Wikia\Tasks\Tasks\MultiTask();
 						$task->call( 'delete', $data );
 						$submit_id = $task->queue();
-
-						Wikia::log( __METHOD__, 'deletecomment', "Added task ($submit_id) for {$data['page']} page" );
```
Comment (macbre) [inline extensions/wikia/ArticleComments/classes/ArticleCommentList.class.php:780]: Pushed tasks are logged by `BaseTask::queue`

File: extensions/wikia/UserProfilePageV3/Masthead.class.php [hunk 2]
```diff
@@ -485,9 +485,7 @@ public function removeFile( $addLog = true ) {
 
 		$result = $this->doRemoveFile();
 
-		if ( $result === false ) {
-			Wikia::log( __METHOD__, false, 'cannot remove avatar - ' . $this->getLocalPath() );
-		} else {
```
Comment (macbre) [inline extensions/wikia/UserProfilePageV3/Masthead.class.php:490]: Errors are logged by a client called in `doRemoveFile`

File: includes/parser/Parser.php [hunk 3]
```diff
@@ -4029,9 +4029,6 @@ function fetchScaryTemplateMaybeFromCache( $url ) {
 
 		$text = Http::get( $url );
 		if ( !$text ) {
-			# wikia start
-			Wikia::log(__METHOD__, false, "Scary transclusion failed for <{$url}>");
-			# wikia end
```
Comment (macbre) [inline includes/parser/Parser.php:4034]: `Http::get` logs errors internally

Comment (wladekb) [inline includes/parser/Parser.php:4034]: How about 200 responses with empty body? Should we have them in the logs?

**Comment by wladekb** [comment]
lgtm, left 1 comment

File: includes/parser/Parser.php (hunk 3 above)
Comment (macbre) [inline includes/parser/Parser.php:4034]: Good point. I'll compare logs from `Http::get` and the ones coming from this `Wikia::log` call .

Comment (macbre) [inline includes/parser/Parser.php:4034]: Addressed in fbf827e

**Comment by wladekb** [comment]
lgtm

**Pull request merged**